code/procs/mobprocs/chatprocs.dm

```diff
@@ -426,6 +428,42 @@
 	if (src.get_brain_damage() >= 60)
 		speechverb = pick("says","stutters","mumbles","slurs")
 
+	if (src.singing && src.bioHolder && src.bioHolder.HasEffect("accent_scots"))
+		text = danny_boy(src.danny_index)
```

Comment:
yeah your earlier danny boy var behavior seems to conflict with this, the vars might be unused?

Reply:
Maybe it's clearer if I move it here:
![image](https://user-images.githubusercontent.com/7035767/96355818-e223d180-1131-11eb-8e5f-5dd5c7645284.png)

Reply:
see earlier explanation